### ISSUE TITLE
Bug + Crash when moving Group/Events/Variables (now with repro)

### ISSUE THREAD

**Issue by Wackytoaster** [Scirra/Construct-3-bugs#1720]
## Problem description

Related to https://github.com/Scirra/Construct-3-bugs/issues/1712
but now with repro. Also Groups vanish. I actually noticed this vanishing bug a couple of times before but I was never certain how to reproduce it (or if I´m just going insane :D)

## Attach a .c3p

🖌 
[hmm.c3p.zip](https://github.com/Scirra/Construct-3-bugs/files/2124466/hmm.c3p.zip)

## Steps to reproduce

*There is no need to explain how you made the attached project. Just explain what to do to with it to observe the issue.*

1. Open example c3p
2. Rightclick Group 1
3. Add > new sub-group
4. Keep the selection as it is (Group 1 + the variable and condition)
5. Drag & Drop the **condition/variable** into Group2. (Dragging Group1 instead will correctly show the error message)
6. (Ctrl+z twice and repeat steps 3-5 --> crash)

Take this video instead (unzip it). Shows the vanishing bug aswell as the crash.
[New project - Construct 3 21.06.2018 12_59_26.zip](https://github.com/Scirra/Construct-3-bugs/files/2123292/New.project.-.Construct.3.21.06.2018.12_59_26.zip)

## Observed result

Groups vanishing, also crash

## Expected result

To just move the events/groups

## Affected browsers

* Chrome: yes

## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

Error report information
Type: assertion failure
Message: no sequence number assigned
Stack: Error at Function.ǃW.ǃYdH (https://editor.construct.net/r104/main.js:2:138063) at t (https://editor.construct.net/r104/main.js:2:104027) at window.assert (https://editor.construct.net/r104/main.js:2:104434) at e.ǃYGV (https://editor.construct.net/r104/main.js:63:292396) at ǃYKU (https://editor.construct.net/r104/main.js:63:292494) at Array.sort (native) at window.ǃYbH.ǃYKB (https://editor.construct.net/r104/components/editors/eventSheetView/eventSheetView.js:1:11878) at window.ǃYbH.ǃYmt (https://editor.construct.net/r104/components/editors/eventSheetView/eventSheetView.js:1:26328) at window.ǃYbH.ǃYoS (https://editor.construct.net/r104/components/editors/eventSheetView/eventSheetView.js:1:25676) at window.ǃYbH.ǃYo (https://editor.construct.net/r104/components/editors/eventSheetView/eventSheetView.js:1:21671)
Construct 3 version: r104
URL: https://editor.construct.net/index.html
Date: Thu Jun 21 2018 12:59:54 GMT+0200 (Mitteleuropäische Sommerzeit)
Uptime: 169.8 s

Platform information
Browser: Chrome
Browser version: 67.0.3396.87
Browser engine: Blink
Browser architecture: 64-bit
Context: webapp
Operating system: Windows
Operating system version: 10
Operating system architecture: 64-bit
Device type: desktop
Device pixel ratio: 1.5
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.87 Safari/537.36
C3 release: r104 (stable)
Language setting: en-US

WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: no
Vendor: Google Inc.
Renderer: ANGLE (Intel(R) HD Graphics 530 Direct3D11 vs_5_0 ps_5_0)
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 1024
Extensions: EXT_color_buffer_float, EXT_texture_filter_anisotropic, OES_texture_float_linear, WEBGL_compressed_texture_s3tc, WEBGL_compressed_texture_s3tc_srgb, WEBGL_debug_renderer_info, WEBGL_debug_shaders, WEBGL_lose_context

</details>

**Comment by AshleyScirra** [comment]
Can't reproduce here. When I try to drag events in to the sub-group (Group2), Construct correctly says that events cannot be moved there, because it's moving a group inside of itself, since you have Group1 selected and you're trying to drop it inside the sub-group Group2.

Please try to stick to the steps to reproduce and .c3p file - it's hard to follow a video in real-time, but written steps to reproduce can be followed slowly and carefully while debugging. Can you find a way to reproduce this while providing a repro project?

**Comment by Wackytoaster** [comment]
@AshleyScirra alright then, updated with c3p and step-by-step.

**Comment by AshleyScirra** [comment]
Thanks, think I was dragging the group rather than the event, I can reproduce now.

**Comment by AshleyScirra** [comment]
Should be fixed in the next release. Thanks for the report, it's a good catch.
